manifests/generated/kubevirt-cr.yaml.in

```diff
@@ -14,6 +14,6 @@ spec:
       {{- end}}{{else}} []{{end}}
   customizeComponents: {}
   imagePullPolicy: {{.ImagePullPolicy}}
-  infra:
-    replicas: {{.InfraReplicas}}
+  infra:{{if .InfraReplicas}}
+    replicas: {{.InfraReplicas}}{{end}}
```

Comment:
I admit to not really understand the changes here :)
Can you elaborate? What does the `if .InfraReplicas` refers to?

Thanks!

Reply:
This is generated code from above (yes, generated template, meta!).
`if .InfraReplicas` will be true if this value is non-zero: https://github.com/kubevirt/kubevirt/blob/09445bf84b9213bb44c150b470ec9b88fbe788dd/tools/manifest-templator/manifest-templator.go#L70